src/main.js

```diff
@@ -77,5 +82,10 @@ if (isFirefoxPanel()) {
   };
 } else {
   bootstrap(React, ReactDOM, App, actions, store)
-    .then(conn => onConnect(conn, actions));
+    .then(conn => onConnect(conn, actions))
+    .then(store => {
+      if (typeof store.getState === "function") {
+        window.landingPageStore = store;
+      }
```

Comment:
hmm, I suppose we can make this global in `launchpad/src/index`.

The launchpad will never be used for evil